.buildkite/dagster-buildkite/dagster_buildkite/utils.py

```diff
@@ -176,3 +177,13 @@ def parse_package_version(version_str: str) -> packaging.version.Version:
         parsed_version, packaging.version.Version
     ), f"Found LegacyVersion: {version_str}"
     return parsed_version
+
+
+def get_changed_files():
+    paths = (
+        subprocess.check_output(["git", "diff", "origin/master...HEAD", "--name-only"])
```

Comment:
are we certain about these commit targets of `origin/master` and `HEAD` ? I've seen issues with our current change detection acting out of line with expectations. I never had the chance to try to dig in and debug but by suspicion is that it has to do with this.

Reply:
I just lifted the same incantation that we use elsewhere. Didn’t realize there were doubts about it. I’ll look into it in more detail tomorrow.